website/source/docs/acl/auth-methods/kubernetes.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "docs"
+page_title: "Kubernetes Auth Method"
+sidebar_current: "docs-acl-auth-methods-kubernetes"
+description: |-
+  The kubernetes auth method type allows for a Kubernetes Service Account Token to be used to authenticate to Consul. This method of authentication makes it easy to introduce a Consul token into a Kubernetes Pod.
+---
+
+-> **1.5.0+:**  This guide only applies in Consul versions 1.5.0 and newer.
+
+# Kubernetes Auth Method
+
+The `kubernetes` auth method type allows for a Kubernetes Service Account Token
+to be used to authenticate to Consul. This method of authentication makes it
+easy to introduce a Consul token into a Kubernetes Pod.
+
+## Config Parameters
+
+The following auth method [`Config`](/api/acl/auth-methods.html#config)
+parameters are required to properly configure an auth method of type
+`kubernetes`:
+
+- `Host` `(string: <required>)` - Must be a host string, a host:port pair, or a
+  URL to the base of the Kubernetes API server. 
+
+- `CACert` `(string: <required>)` - PEM encoded CA cert for use by the TLS
+  client used to talk with the Kubernetes API. NOTE: Every line must end with a
+  newline (`\n`).
+
+- `ServiceAccountJWT` `(string: <required>)` - A Service Account Token (JWT)
```

Comment:
kait: Capitalization? Is there a way to be a little more clear that JWT actually means JSON web token?